### ISSUE TITLE
Skin lesions on limbs

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9945]
**Preferred term label:**
Skin lesions on limbs

**Synonyms**
appendicular

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Buruli ulcer

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
adding
Appendicular location
HP:0020069
